inst/shiny-server/server/ERT_summary.R

```diff
@@ -2,8 +2,9 @@
 runtime_summary_condensed <- reactive({
   data <- DATA()
   req(length(data) > 0)
-  fall <- get_funvals(data)
-  df <- get_FV_overview(data, algorithm = input$RTSummary.Overview.Algid)
+  # fall <- get_funvals(data)
+  data <- subset(data, ID %in% input$RTSummary.Overview.Algid)
```

Comment:
Would it make sense to rename `RTSummary.Overview.Algid` to `RTSummary.Overview.ID`? Or, it is really intended as the algorithm name?